source/1_engine/rex_engine/include/rex_engine/diagnostics/logging/logger_config.h

```diff
@@ -6,6 +6,6 @@ namespace rex
 {
   namespace diagnostics
   {
-    void load_log_levels(const rsl::vector<rsl::string_view>& args);
+    void init();
```

Comment:
This is fixed in the other branch right?

Reply:
yes